libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator+Internal.h

```diff
@@ -48,6 +48,12 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong ,nullable) SFOAuthCredentials *spAppCredentials;
 @property (nonatomic, weak, nullable) SFSDKAuthSession *authSession;
 
+/// For Salesforce Identity UI Bridge  API support, an overriding front door bridge URL to use in place of the default inital URL.
+@property (nonatomic, strong, nullable) NSURL *overrideWithfrontDoorBridgeUrl;
```

Comment:
There might be other ways we could do this.  It seemed to me this would follow our existing patterns and minimize our code footprint and risk for now.  These two new `SFOAuthCoordinator` properties give us all the support we need for QR code login, plus we have new convenience parameters in overloads to existing methods to pass them through.

Take a look over this and let me know if other ideas come to mind.